packages/zowe-explorer/__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -104,25 +104,100 @@ describe("Shared Utils Unit Tests - Function node.labelRefresh()", () => {
     });
 });
 
-describe("Shared Utils Unit Tests - Function refreshTree()", () => {
-    async function createBlockMocks(globalMocks) {
-        const newMocks = {
-            log: Logger.getAppLogger(),
-            imperativeProfile: createIProfile(),
-            testDatasetSessionNode: null,
-        };
-        newMocks.testDatasetSessionNode = createDatasetSessionNode(globalMocks.session, newMocks.imperativeProfile);
-        return newMocks;
-    }
-    it("should pass in appropriate profile type when getting all profiles to compare with session node", async () => {
-        const globalMocks = await createGlobalMocks();
-        const blockMocks = await createBlockMocks(globalMocks);
-        globalMocks.mockGetInstance.mockReturnValue(blockMocks.imperativeProfile);
-        const getProfilesSpy = jest.spyOn(Profiles.getInstance(), "getProfiles");
-
-        utils.refreshTree(blockMocks.testDatasetSessionNode);
+describe("syncSession shared util function", () => {
+    const serviceProfileName = "test";
+    const serviceProfileType = "zosmf";
+    const serviceProfile = {
+        name: serviceProfileName,
+        profile: {},
+    };
 
-        expect(getProfilesSpy).toBeCalledWith(globalMocks.profileOne.type);
+    // we don't really care about the node type here, can be anything
+    const sessionNode = new ZoweDatasetNode(
+        "profile session",
+        vscode.TreeItemCollapsibleState.Collapsed,
+        null,
+        undefined,
+        undefined,
+        undefined,
+        {
+            profile: serviceProfile,
+            type: serviceProfileType,
+            message: "",
+            failNotFound: true,
+        }
+    );
```

Comment:
I remember moving away from defining test items in this way and using the global mocks instead for more consistency. 
I'm fine with it personally in this unit test since it is a small function, but I would love to hear @katelynienaber 's thoughts on it.

Reply:
Yes, thank you for noticing this :D

The goal is to remove these big, repetitive mock declarations all over the place. You can find some mock creation functions in folder `zowe-explorer/__mocks__/mockCreators`, ex. I would use this one here: https://github.com/zowe/vscode-extension-for-zowe/blob/00f2ec21e95713bb2bfa2311fb23c6e4c4d3f904/packages/zowe-explorer/__mocks__/mockCreators/datasets.ts#L19

You can ofc change individual properties on your node after you use the mock creation function, but using it will make the declaration a one-liner.

Also you can check in the `createGlobalMocks` or `createBlockMocks` functions, to see if some session node has already been created there for you to use. The mocks created in those functions are recreated after every it() block, so you can be sure you won't affect later tests by changing the properties of any nodes declared there.

Reply:
Thank you for this point @katelynienaber , I will take a look and try to reuse already defined mock nodes here.